src/frontend-scripts/components/section-main/Gamechat.jsx

```diff
@@ -674,6 +674,9 @@ class Gamechat extends React.Component {
 				</section>
 				<section className={this.state.claim ? 'claim-container active' : 'claim-container'}>
 					{(() => {
+						if (gameInfo.gameState.isCompleted) {
+							this.state.claim = '';
+						}
 						if (this.state.claim) {

```

Comment:
React doesn't really work that way 😉 What you want here is on line 680 to be something like `if (this.state.claim && !gameInfo.gameState.isCompleted)`